src/Drive.php

```diff
@@ -290,7 +290,25 @@ public function getResources(string $path = "/"): array
         $resources = [];
         $webDavClient = $this->createWebDavClient();
         try {
-            $responses = $webDavClient->propFind(rawurlencode(ltrim($path, "/")), [], 1);
+            $properties = [
+                '{http://owncloud.org/ns}id',
+                '{http://owncloud.org/ns}fileid',
+                '{http://owncloud.org/ns}spaceid',
+                '{http://owncloud.org/ns}file-parent',
+                '{http://owncloud.org/ns}name',
+                '{DAV:}getetag',
+                '{http://owncloud.org/ns}permissions',
+                '{DAV:}resourcetype',
+                '{http://owncloud.org/ns}size',
+                '{DAV:}getcontentlength',
+                '{DAV:}getlastmodified',
+                '{http://owncloud.org/ns}tags',
+                '{http://owncloud.org/ns}favorite',
+                '{http://owncloud.org/ns}privatelink',
+                '{DAV:}getcontenttype',
+                '{http://owncloud.org/ns}checksums'
+            ];
```

Comment:
do we need to send all properties?

Reply:
yes, those are the properties that we currently support, all except `'{http://owncloud.org/ns}privatelink'` come by default with a PROPFIND without requesting a property, so if we want to keep backward compatibility and still support all the properties, they need all to be listed here